src/sound_openal.cpp

```diff
@@ -265,6 +284,46 @@ struct PlayingSound
 	bool loop;
 };
 
+class SoundManagerGlobal
+{
+public:
+	unique_ptr_alcdevice  m_device;
+	unique_ptr_alccontext m_context;
+public:
+	SoundManagerGlobal() :
+		m_device(NULL, delete_alcdevice),
+		m_context(NULL, delete_alccontext)
+	{
+		if (! (m_device = unique_ptr_alcdevice(alcOpenDevice(NULL), delete_alcdevice)))
+			throw std::runtime_error("Audio: Global Initialization: Device Open");
+
+		if (! (m_context = unique_ptr_alccontext(alcCreateContext(m_device.get(), NULL), delete_alccontext)))
```

Comment:
Line too long (> 80 characters). Could you please also remove the spaces after each unary operator (`!`) due our code style guidelines?

Reply:
Done (also in the site about 3 lines below). Readability of line split is worse there though IMO hehe.